packages/reputation-miner/test/MaliciousReputationMinerClaimNoOriginReputation.js

```diff
@@ -1,53 +1,47 @@
       this.replacementAmount = reputationValue.mul(new BN(-1));
-      console.log(this.replacementAmount.toString())
     }
     await super.addSingleReputationUpdate(updateNumber, repCycle, blockNumber, checkForReplacement)
+    if (updateNumber.toNumber() === this.entryToFalsify){
 
-    // Set the origin skill key
-    const logEntryNumber = this.getLogEntryNumberForLogUpdateNumber(updateNumber.sub(this.nReputationsBeforeLatestLog));
-    const logEntry = await repCycle.getReputationUpdateLogEntry(logEntryNumber);
-    const originSkillUpdateNumber = logEntry.nUpdates.add(logEntry.nPreviousUpdates).add(this.nReputationsBeforeLatestLog).sub(1);
-    const originReputationKey = await this.getKeyForUpdateNumber(originSkillUpdateNumber);
-    this.justificationHashes[ReputationMinerTestWrapper.getHexString(updateNumber, 64)].originReputationProof.key = originReputationKey;
-
-    // Set the child skill key
-    const relativeUpdateNumber = updateNumber.sub(this.nReputationsBeforeLatestLog).sub(logEntry.nPreviousUpdates);
-    const {nUpdates} = logEntry;
-    const [nParents] = await this.colonyNetwork.getSkill(logEntry.skillId);
-    const nChildUpdates = nUpdates.div(2).sub(1).sub(nParents);
-    let childKey;
-    if (relativeUpdateNumber.lt(nChildUpdates)) {
-      const childSkillUpdateNumber = updateNumber.add(nUpdates.div(2));
-      childKey = await this.getKeyForUpdateNumber(childSkillUpdateNumber);
-    } else {
-      childKey = await this.getKeyForUpdateNumber(updateNumber);
+      // Set the origin skill key
+      const logEntryNumber = await this.getLogEntryNumberForLogUpdateNumber(updateNumber.sub(this.nReputationsBeforeLatestLog));
+      const logEntry = await repCycle.getReputationUpdateLogEntry(logEntryNumber);
+      const originSkillUpdateNumber = logEntry.nUpdates.add(logEntry.nPreviousUpdates).add(this.nReputationsBeforeLatestLog).sub(1);
+      const originReputationKey = await this.getKeyForUpdateNumber(originSkillUpdateNumber);
+      this.justificationHashes[ReputationMinerTestWrapper.getHexString(updateNumber, 64)].originReputationProof.key = originReputationKey;
+      // Set the child skill key
+      const relativeUpdateNumber = updateNumber.sub(this.nReputationsBeforeLatestLog).sub(logEntry.nPreviousUpdates);
+      const {nUpdates} = logEntry;
+      const [nParents] = await this.colonyNetwork.getSkill(logEntry.skillId);
+      const nChildUpdates = nUpdates.div(2).sub(1).sub(nParents);
+      let childKey;
+      if (relativeUpdateNumber.lt(nChildUpdates)) {
+        const childSkillUpdateNumber = updateNumber.add(nUpdates.div(2));
+        childKey = await this.getKeyForUpdateNumber(childSkillUpdateNumber);
+      } else {
+        childKey = await this.getKeyForUpdateNumber(updateNumber);
+      }
+      this.justificationHashes[ReputationMinerTestWrapper.getHexString(updateNumber, 64)].childReputationProof = 
+        await this.getReputationProofObject(childKey);
```

Comment:
It's not immediately obvious how this client mades an existing origin reputation disappear. Maybe add some comments? When debugging to find out more I got this out for the bad client's final set of 6 updates (that are relevant for this), see below. 
```
updateNumber 34
key 0xd7474abde95761bc61742a8bb2269332d9dc81e700000000000000000000000000000000000000000000000000000000000000050000000000000000000000000000000000000000
amount 0
originReputationKey 0xd7474abde95761bc61742a8bb2269332d9dc81e700000000000000000000000000000000000000000000000000000000000000049f485401a3c22529ab6ea15e2ebd5a8ca54a5430
childKey 0xd7474abde95761bc61742a8bb2269332d9dc81e700000000000000000000000000000000000000000000000000000000000000059f485401a3c22529ab6ea15e2ebd5a8ca54a5430
updateNumber 35
key 0xd7474abde95761bc61742a8bb2269332d9dc81e700000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000
amount -500000000000
updateNumber 36
key 0xd7474abde95761bc61742a8bb2269332d9dc81e700000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000
amount -500000000000
updateNumber 37
key 0xd7474abde95761bc61742a8bb2269332d9dc81e700000000000000000000000000000000000000000000000000000000000000059f485401a3c22529ab6ea15e2ebd5a8ca54a5430
amount -500000000000
updateNumber 38
key 0xd7474abde95761bc61742a8bb2269332d9dc81e700000000000000000000000000000000000000000000000000000000000000019f485401a3c22529ab6ea15e2ebd5a8ca54a5430
amount -500000000000
updateNumber 39
key 0xd7474abde95761bc61742a8bb2269332d9dc81e700000000000000000000000000000000000000000000000000000000000000049f485401a3c22529ab6ea15e2ebd5a8ca54a5430
amount -500000000000
```